app/views/shared/formatter/_version_diff.html.haml

```diff
@@ -0,0 +1,47 @@
+- case object.event
+- when 'create'
+  - case object.item
+  - when Approval
+    = object.item.user.full_name
+    was added as an approver.
+  - when Attachment
+    Uploaded
+    = succeed '.' do
+      %code
+        = object.item.file_file_name
+  - when Comment
+    Commented:
+    = surround '"', '"' do
+      = object.item.comment_text
+  - when Observation
+    = object.item.user.full_name
+    was added as an observer.
+- when 'update'
+  %ul
+    - value.each do |change|
+      - next if %w(created_at updated_at).include?(change[1])
+      %li
+        - case change[0]
+        - when '+' # added
+          %code
+            = change[1]
+          was set to
+          = succeed '.' do
+            %code
+              = diff_val(change[2])
+        - when '~' # modified
+          %code
+            = change[1]
+          was changed from
+          %code
+            = diff_val(change[2])
+          to
+          = succeed '.' do
+            %code
+              = diff_val(change[3])
+        - when '-' # removed
+          %code
+            = change[1]
+          was removed.
+        - else
+          = change.inspect
```

Comment:
There's a lot of logic in this display. What do you think about moving it into methods, particularly since we've override the version class?

Also: `change` is apparently an array, but it's unclear what it's positional values correspond to